src/lib/tree/Types.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.19;
 
-type NamespaceID is bytes8;
+type NamespaceID is bytes29;
```

Comment:
Hmmm. Should this instead be a struct of a `bytes1` and `bytes28` https://github.com/celestiaorg/celestia-app/blob/3e6d22cfde678efac4f7c078228e9f294a7a9b8d/specs/src/specs/namespace.md#overview? The actual namespace ID isn't just one field. There isn't any code in the QGB proper that cares, but perhaps downstream contracts might care (e.g. to ensure their data was posted to the correct namespace).

Doing that would increase costs though.

Reply:
Does it make sense to handle this in a separate PR? I can create an issue for it and then make the change